modules/auxiliary/gather/browser_info.rb

```diff
@@ -74,7 +74,7 @@ def print_target_info(cli, target_info)
 
   def on_request_exploit(cli, req, target_info)
     print_target_info(cli, target_info)
-    send_not_found(cli)
+    send_response(cli, '')
```

Comment:
The original code sends a 404. The new code sends an empty 200 OK. This avoids an oracle situation I suppose.